apps/src/turtle/SaveImageButton.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import msg from '@cdo/locale';
```

Comment:
nit/question: i usually see this import referred to as `i18n`, but we do refer to it as `msg` often. do you know if there's a preferred version? i'd like for us to be consistent

Reply:
![image](https://user-images.githubusercontent.com/8787187/111383758-ea8f2280-8665-11eb-986d-a1e4de6e5622.png)

Looks like we do both but mostly `i18n`. I'll update.